lib/overhang.js

```diff
@@ -44,6 +44,9 @@ $.fn.overhang = function (arguments) {
     easing       : "easeOutBounce",
     html         : false,
     overlay      : false,
+    multiline    : false,
+    textWidth    : "250px",
```

Comment:
Changing the name of "textWidth" and "textHeight" to "textarea___" would be better as these variables don't apply to the input field.